.github/workflows/pr-all-services.yml

```diff
@@ -0,0 +1,74 @@
+name: All Services Tests
+
+on:
+  pull_request:
+
+jobs:
+  leia-tests:
+    runs-on: ${{ matrix.os }}
+    env:
+      TERM: xterm
+    strategy:
+      matrix:
+        leia-tests:
+          - examples/all-services
+        lando-versions:
+          - edge
+        os:
+          - ubuntu-22.04
+        node-version:
+          - '16'
+
+    steps:
+      # Install deps and cache
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Install node ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: yarn
+      - name: Install Yarn dependencies
+        run: yarn install --prefer-offline --frozen-lockfile
+
+      # This block should eventually become use lando/actions-hyperdrive@v2
+      - name: Verify Docker dependencies
+        run: |
+          docker --version | grep "20.10."
+          docker-compose --version | grep "1.29."
+      - name: Grab latest edge Lando CLI
+        run: |
+          sudo curl -fsSL -o /usr/local/bin/lando "https://files.lando.dev/cli/lando-linux-x64-${{ matrix.lando-versions }}"
+          sudo chmod +x /usr/local/bin/lando
+      - name: Find and Replace
+        uses: jacobtomlinson/gha-find-replace@v3
+        with:
+          find: "lagoon/lagoon"
+          replace: ${{ github.event.repository.name }}/${{ github.event.repository.name }}
+          include: "actions-lando-config.yml"
+          regex: false
```

Comment:
quick note here - we needed to replace the lando-config here, as our fork of the repo isn't in a `lagoon` repo - so this allows the config to pick up the correct repo name